### ISSUE TITLE
NVDA/Chrome HTML "region" reading parent and child element text

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#32]
This is an issue in reference to #25 which is related to the ["region" branch](https://github.com/fizzstudio/svg-aria-test/tree/aria_region).

<img width="1140" alt="Screenshot of NVDA in test environment. Speech viewer for NVDA is focusing on one element and reads A graphical display of data Each element contains different information  region    Rect and Circle  grouping    graphic    3 Rect" src="https://user-images.githubusercontent.com/98043076/152582534-784363b2-0e34-47d2-82da-424c09ddd041.png">

### Issue Summary
NVDA reads the HTML parent and first child grouping of the region when navigating to the starting graphic element. Similarly, the next grouping also does not work properly. 

### Video of issue
Example starts at 00:00:38 and 00:00:46
https://user-images.githubusercontent.com/98043076/152582325-7efb28fc-9cdb-436e-9629-182d0cdf8477.mp4

### Steps to reproduce
Use NVDA "H" key to navigate to heading "html examples." Use the down arrow key to navigate into the HTML space. There is only one step (we would expect at least nine separate elements to navigate.)

### Expected behavior
We aren't actually testing HTML in this project, but if we are comparing the SVG to HTML elements in how they perform we need to note the issues found. 

### Technical Details
Chrome Version 97.0.4692.99
NVDA 2021.3.1 
Windows 11 Build 22000.438
